core/tests/protractor_utils/CreatorDashboardPage.js

```diff
@@ -69,14 +69,17 @@ var CreatorDashboardPage = function() {
   // Returns all exploration card elements with the given name.
   var _getExplorationElements = async function(explorationTitle) {
     await waitFor.visibilityOf(explorationDashboardCard);
-    return await allExplorationCards.filter(async function(tile) {
-      var text = await tile.getText();
+    var filteredExplorationCards = [];
+    for (i = 0; i < allExplorationCards.length; i++) {
+      var text = await allExplorationCards[i].getText();
       // Tile text contains title, possibly followed by newline and text.
-      return (
-        text.startsWith(explorationTitle + '\n') ||
-        text === explorationTitle
-      );
-    });
+      if (
+        text.startsWith(explorationTitle + '\n') || text === explorationTitle
+      ) {
+        filteredExplorationCards.push(text);
+      }
+    }
+    return filteredExplorationCards;
```

Comment:
Why is this changed?

Reply:
Hey @vojtechjelinek, this is because of the issue https://github.com/oppia/oppia/issues/11496. 

We are not allowing to use .filter() function anymore

Reply:
But that should be done in a different PR.

Reply:
Yeah, that was in another separate PR, don't know why it got accidentally included also in this PR